### ISSUE TITLE
`watchmedo automatic-restart` doesn't terminate child processes when it is terminated

### ISSUE THREAD

**Issue by Varriount** [gorakhargosh/watchdog#609]
Currently, `watchmedo automatic-restart` creates its command in a new process group. Because of this, if `watchmedo` is terminated (for example, by ctrl+c) the running command will not be terminated.

I can add a new flag the prevents putting the process into a new process group (though I'm wondering why you would want to in the first place). Would this be accepted?

**Comment by BoboTiG** [comment]
Related to #225?

**Comment by Varriount** [comment]
@BoboTiG Yes, although I don't know why the creator of the original issue had that problem. By default on *nix systems, signals propagate from a parent process to all children in the same process group.

**Comment by hholst80** [comment]
Maybe they are running watchmedo as PID 1 in a container?
